project/levels/battles/battle/battle_grid/battle_grid.gd

```diff
@@ -17,9 +19,9 @@ var fly_navigation: NavigationLayer
 var walk_navigation: NavigationLayer
 
 
-func load(battlefield: Battlefield, layout: GridObjectLayout):
+func load(battlefield: Battlefield, layout: GridObjectLayout, teams: Array[Team]):
```

Comment:
We can change this later, but since load is a global function, it may be best to not override it in this functions scope.